modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -7428,15 +7428,9 @@ private Map<Locale, String> _checkFriendlyURLMap(
 		Locale defaultLocale, Map<Locale, String> friendlyURLMap,
 		Map<Locale, String> titleMap) {
 
-		for (Map.Entry<Locale, String> friendlyURL :
-				friendlyURLMap.entrySet()) {
-
-			if (Validator.isNotNull(friendlyURL.getValue())) {
-				return friendlyURLMap;
-			}
-		}
-
-		return HashMapBuilder.put(
+		return HashMapBuilder.putAll(
+			friendlyURLMap
+		).put(
```

Comment:
This is not correct. If there's a value for the default locale you'll overwrite it.

Reply:
oh I see, mu fault